news/5128.hooks.rst

```diff
@@ -0,0 +1 @@
+Updated a hook for sqlalchemy, by adding a hidden import.
```

Comment:
This is a bit generic - please clarify what the hidden import is, and why it needed adding.

Reply:
```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``pysqlite2``, ``MySQLdb`` and ``psycopg2`` as hidden imports.
```